tests/features/group.feature

```diff
@@ -41,6 +41,40 @@ Feature: Subcommand: dab group
 		Executing four entrypoint start
 		"""
 
+	Scenario: Can group repositories fail fast
+		Given I successfully run `dab repo add successOne https://github.com/Nekroze/dotfiles.git`
+		And I run `dab repo entrypoint create successOne start`
+		Given I successfully run `dab repo add failure https://github.com/Nekroze/dotfiles.git`
```

Comment:
I think the linter will want this block to only have one given iirc.

Reply:
Yea linter didn't barf but i'll update it to be consistent.